src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -198,11 +216,12 @@ private function getOrderToolbarButtons(): array
      * @return RedirectResponse
      */
     #[AdminSecurity("is_granted('create', request.get('_legacy_controller'))")]
-    public function placeAction(Request $request)
-    {
+    public function placeAction(
```

Comment:
I don't really know what to do about the phpstan error on the 
`if ($result->getIdentifiableObjectId() instanceof OrderId)`